waddrmgr/manager_test.go

```diff
@@ -1067,6 +1068,10 @@ func testMarkUsed(tc *testContext) bool {
 		},
 	}
 
+	if !doScript {
+		tests = tests[0:1]
```

Comment:
Alternatively, we can add a new `bool` to the table tests above, and skip them below based on that bool.

Reply:
Fixed.